packages/hardhat/contracts/SignatureNFT.sol

```diff
@@ -6,7 +6,7 @@ import { Counters } from "@openzeppelin/contracts/utils/Counters.sol";
 
 contract SignatureNFT is ERC721Tradable {
     using Counters for Counters.Counter;
-    Counters.Counter private _tokenIdCounter;
+    Counters.Counter public tokenIdCounter;
```

Comment:
open to suggestions here. What I need is a way to know what the _next_ mintable tokenId will be

Reply:
It is conceivable that you could get this from our subgraph by looking at `data.signatureNFTs(data.signatureNFTs.length() - 1).id`, but I don't think making this public is an issue either. 

However, if you make this change, you will also need to change L52 in this contract. I would prefer not to merge here while the contract is broken.

Reply:
yeah i did change it...
so it's all right if I make it public right?